auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/domain/User.java

```diff
@@ -0,0 +1,157 @@
+package com.atlantbh.auctionappbackend.domain;
+
+import javax.persistence.*;
```

Comment:
Avoid wildcard imports.

Reply:
OK, I'll fix this, and keep it in mind for the future.